src/crons/targets.cron.ts

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable max-len */
+import cron from 'node-cron';
+import { Service } from 'typedi';
+import { TargetsService } from '@services/targets.service';
+import { SocketService } from '@services/socket.service';
+
+@Service()
+export class TargetCron {
+  constructor( private targetsService: TargetsService, private socketService: SocketService ) { }
+
+  async checkTargetsMatch( ): Promise<void> {
+    const newTargets = await this.targetsService.anyNewTargets( );
+    if ( newTargets ) {
+      let targetsMatch = [];
+      const targetsByTopic = await this.targetsService.showTargetsByTopic( );
+      Object.keys(targetsByTopic).forEach( ( topicId: string ) => {
+        const targetsMatched = this.targetsService.getTargetsMatched( targetsByTopic[topicId] );
+        targetsMatch = [...targetsMatch, targetsMatched].flat();
+      });
+      targetsMatch.forEach( ( targets: any ) => {
```

Comment:
We can avoid the usage of `any` by creating an interface of a target and then generating a type for the array